examples/design-system/README.md

```diff
@@ -53,44 +52,49 @@ This example sets up your `.gitignore` to exclude all generated files, other fol
 
 ### Compilation
 
-To make the core library code work across all browsers, we need to compile the raw TypeScript and React code to plain JavaScript. We can accomplish this with `tsup`, which uses `esbuild` to greatly improve performance.
+To make the ui library code work across all browsers, we need to compile the raw TypeScript and React code to plain JavaScript. We can accomplish this with `tsup`, which uses `esbuild` to greatly improve performance.
 
 Running `pnpm build` from the root of the Turborepo will run the `build` command defined in each package's `package.json` file. Turborepo runs each `build` in parallel and caches & hashes the output to speed up future builds.
 
-For `acme-core`, the `build` command is the following:
+For `@acme/ui`, the `build` command is equivalent to the following:
 
 ```bash
-tsup src/index.tsx --format esm,cjs --dts --external react
+tsup src/*.tsx --format esm,cjs --dts --external react
 ```
 
-`tsup` compiles `src/index.tsx`, which exports all of the components in the design system, into both ES Modules and CommonJS formats as well as their TypeScript types. The `package.json` for `acme-core` then instructs the consumer to select the correct format:
+`tsup` compiles all of the components in the design system individually, into both ES Modules and CommonJS formats as well as their TypeScript types. The `package.json` for `@acme/ui` then instructs the consumer to select the correct format:
 
-```json:acme-core/package.json
+```json:ui/package.json
 {
-  "name": "@acme/core",
+  "name": "@acme/ui",
   "version": "0.0.0",
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
   "sideEffects": false,
+  "exports":{
+    "./button": {
+      "types": "./src/button.tsx",
+      "import": "./dist/button.mjs",
+      "require": "./dist/button.js"
+    }
+  }
 }
 ```
 
-Run `pnpm build` to confirm compilation is working correctly. You should see a folder `acme-core/dist` which contains the compiled output.
+Run `pnpm build` to confirm compilation is working correctly. You should see a folder `ui/dist` which contains the compiled output.
 
 ```bash
-acme-core
+ui
 └── dist
-    ├── index.d.ts  <-- Types
-    ├── index.js    <-- CommonJS version
-    └── index.mjs   <-- ES Modules version
+    ├── button.d.ts  <-- Types
+    ├── button.js    <-- CommonJS version
+    ├── button.mjs   <-- ES Modules version
+    └── button.mts   <-- ES Modules version with Types
```

Comment:
```suggestion
    └── button.d.mts   <-- ES Modules version with Types
```

as it turns out, the file that's actually generated is `button.d.mts`: was this intentional?  seems like not because it's exactly the same as the `button.d.ts`

Reply:
@dimitropoulos dear Dimitri, thank you for your review. Yes you're right. I'll fix it right now.